src/main/java/alpha/UI.java

```diff
@@ -80,39 +71,29 @@ public void unmark(Task task) {
      * @param task Task that is being marked.
      * @param size Remaining size of the task list.
      */
-    public void delete(Task task, int size) {
-        System.out.println(LINE_SEPARATOR + "\n" +
-                "Noted. I've removed this task:\n" + "  " + task + "\n Now You have " + size +
-                " tasks in the list." + "\n" + LINE_SEPARATOR);
+    public String delete(Task task, int size) {
+        return "Noted. I've removed this task:\n" + "  " + task + "\n Now You have " + size +
+                " tasks in the list.";
```

Comment:
Maybe can move the '+' sign from the end of a line to the start of the next line? Also, make the split such that the two halves are not too different in length.

So can rewrite as something like this:
```ruby
return "Noted. I've removed this task:\n" + "  " + task
        + "\n Now You have " + size + " tasks in the list.";
```